website/www/site/layouts/partials/pillars/pillars-social.html

```diff
@@ -0,0 +1,39 @@
+{{/*
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+   http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License. See accompanying LICENSE file.
+*/}}
+
+<div class="pillars-social-icons">
+  <div class="pillars-github-icon">
+    <a href="https://github.com/apache/beam" target="_blank">
+      {{ with resources.Get .github }}
+        {{ .Content | safeHTML }}
+      {{ end }}
+    </a>
+  </div>
+  <div class="pillars-youtube-icon">
+    <a href="https://www.youtube.com/channel/UChNnb_YO_7B0HlW6FhAXZZQ" target="_blank">
```

Comment:
Would it be possible for these to be completely configurable like the main pillars? As it stands the data about each is spread between this file and index.html

Reply:
Thanks for your suggestion. I refactored it!